.github/workflows/cve-scan.yaml

```diff
@@ -0,0 +1,31 @@
+name: Docker Image Scan
+on:
+  push:
+    branches:
+      - 'master'
+    tags:
+      - 'v*.*.*'
+  pull_request:
+    branches:
+      - 'master'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Build images
+        shell: bash
+        run: |
+          make docker
+      - name: Scan image Sqlite
+        uses: anchore/scan-action@v3
+        with:
+          image: oryd/keto:latest
```

Comment:
```suggestion
          image: oryd/keto:latest-sqlite
```

from the Makefille:
```Makefile
.PHONY: docker
docker:
		docker build -f .docker/Dockerfile-build -t oryd/kratos:latest-sqlite .
```

Reply:
Oh sorry, that was Kratos. Wrong IDE window....

Reply:
no worries, kratos is next :D